apps/dev/pages/api/auth/[...nextauth].ts

```diff
@@ -91,6 +92,13 @@ export const authOptions: NextAuthOptions = {
       clientId: process.env.GITHUB_ID,
       clientSecret: process.env.GITHUB_SECRET,
     }),
+    ShopifyProvider({
+      clientId: process.env.SHOPIFY_ID,
+      clientSecret: process.env.SHOPIFY_SECRET,
+      shop: "unique-master", // Don't include .myshopify.com
+      scope: "read_products,write_products,read_orders,write_orders",
+      apiVersion: "2022-01",
```

Comment:
```suggestion
```

These should not be necessary, scope is also not a top-level option anymore

Reply:
For every Shopify apps scope is a must, each app has different features and according to that the app request Shopify with the required scopes.

The scope should be received from the app. Is there any way to receive scope from the user if the scope is also not a top-level option anymore?